src/kili/services/plugins/upload.py

```diff
@@ -118,14 +138,65 @@ def _create_plugin_runner(self):
         result = self.auth.client.execute(GQL_CREATE_PLUGIN_RUNNER, variables)
         return format_result("data", result)
 
+    def _check_plugin_runner_status(self, update=False):
+        """
+        Check the status of a plugin's runner until it is active
+        """
+
+        logger = self.get_logger()
+
+        action = "updated" if update else "created"
+
+        logger.info(f"Plugin is being {action}... This should take approximately 3 minutes.")
+
+        n_tries = 0
+        status = None
+
+        logger = self.get_logger("INFO")
+
+        while n_tries < NUMBER_TRIES_RUNNER_STATUS:
+            status = self._get_plugin_runner_status()
+
+            if status == "ACTIVE":
+                break
+
+            logger.info(f"Status : {status}...")
+
+            n_tries += 1
+            time.sleep(15)
+
+        if status == "DEPLOYING" and n_tries == 20:
+            raise Exception("Limit of number of tries exceeded.")
+
+        if status != "ACTIVE":
+            raise Exception("There was some error during the creation of the plugin.")
```

Comment:
Same here, should we tell them to try again ?

Reply:
Updated the message saying to use `update_plugin`, since the plugin was already created, just trying again would not work